update readme (#1577)
nrwl/nx: README.md

<div align="center">

> [![Build Status](https://travis-ci.org/nrwl/nx.svg?branch=master)](https://travis-ci.org/nrwl/nx)

Reviewer: Is this removed because the number isn't entirely representative of Nx's usage?

Author: yup, we point at the deprecated package

# What is Nx?

🔎 **Extensible Dev Tools for Monorepos.**

## Nx Helps You

### Use Modern Tools

Using Nx, you can add TypeScript, Cypress, Jest, Prettier, and Nest into your dev workflow. Nx sets up these tools and allows you to use them seamlessly. Nx fully integrates with the other modern tools you already use and love.

### Build Full-Stack Applications

> With Nx, you can build full-stack applications using modern frameworks. You can share code between the frontend and the backend. And you can use the same `build/test/serve` commands to power whole dev experience.

Reviewer: ```suggestion
With Nx, you can build full-stack applications using modern frameworks. You can share code between the frontend and the backend. And you can use the same `build/test/serve` commands throughout the whole dev experience.
```

Author: fixed

> ### Build like Google, Facebook, and Microsoft

Reviewer: Build > Develop?

It's also the same verb as the previous point

Author: fixed

With Nx, you can develop multiple full-stack applications holistically and share code between them all in the same workspace. Nx provides advanced tools which help you scale your enterprise development. Nx also helps enforce your organization’s standards and community best practices.

> # Getting Started

Reviewer: We should recommend installing `@nrwl/cli` globally if we are going to use the `nx` cli in our documentation.

Author: `create-nx-workspace` will show a message recommending you to install it globally. Do you think it is enough?

Reviewer: It's worth mentioning in the README + our guides as well.

yarn create nx-workspace myworkspace
```

> ### Adding Nx to an Existing Angular CLI workspace

Reviewer: ```suggestion
## Adding Nx to an Existing Angular CLI workspace
```

I think adding to an existing angular is separate from creating a workspace

Author: I think the intent is the same: create an nx workspace. I just want to say "you could also add it instead of creating a new workspace"

Reviewer: Yeah, I think it falls under the broader scope of getting started as 2 separate options.

If you are an Angular user, you can also add Nx to your existing Angular CLI project by running:

```bash
ng add @nrwl/workspace
```

## Creating First Application

By default, an Nx workspace starts blank. There are no applications to build, serve, and test. To create one, you need to add capabilities to the workspace.

> **If you want to add an Angular app, you can run:**

Reviewer 2: ```suggestion
**To add an Angular app, run:**
```

Author: fixed

```bash
yarn add @nrwl/angular
nx g @nrwl/angular:app myapp # or just "nx g myapp"
```

```bash
npm install --save-dev @nrwl/angular
nx g @nrwl/angular:app myapp # or just "nx g myapp"
```

> **If you want to add a React app, you can run:**

Reviewer 2: Same

```bash
yarn add @nrwl/react
nx g @nrwl/react:app myapp # or just "nx g myapp"
```

```bash
npm install --save-dev @nrwl/react
nx g @nrwl/react:app myapp # or just "nx g myapp"
```

> **If you want to add a web components app, you can run:**

Reviewer 2: Same
